nosql/1.0/_index.md

```diff
@@ -46,12 +46,12 @@ template.delete(Car.class, 1L);
 
 
 * [Jakarta NoSQL Release Plan](https://projects.eclipse.org/projects/ee4j.nosql/governance)
-* [Jakarta NoSQL 1.0 Specification Document](./nosql-1.0.0-M1.pdf) (PDF)
-* [Jakarta NoSQL 1.0 Specification Document](./nosql-1.0.0-M1.html) (HTML)
+* [Jakarta NoSQL 1.0 Specification Document](./nosql-1.0.0.pdf) (PDF)
+* [Jakarta NoSQL 1.0 Specification Document](./nosql-1.0.0.html) (HTML)
 * [Jakarta NoSQL 1.0 Specification Javadoc](./apidocs)
 * Maven coordinates
-  * [jakarta-nosql-api:jakarta.nosql:jar:1.0.0-M1](https://repo1.maven.org/maven2/jakarta/nosql/jakarta.nosql-api/1.0.0-M1/)
-
+  * [jakarta-nosql-api:jakarta.nosql:jar:1.0.0](https://repo1.maven.org/maven2/jakarta/nosql/jakarta.nosql-api/1.0.0)
```

Comment:
I'd expect 1.0.0 to be at least staged (if not in Central), but I see only
![image](https://github.com/user-attachments/assets/486a3e31-2a6c-4147-b310-1f9909f3cb9c)

Reply:
That would be my question: can I release version 1.0.0 even without approval yet?